packages/cli/README.md

```diff
@@ -0,0 +1,50 @@
+# Zowe Native Protocol plug-in for Zowe CLI
+
+The Zowe Native Protocol plug-in for Zowe CLI interacts with the Zowe Native Protocol, a native integration with z/OS systems over SSH, allowing developers to interact with mainframe resources directly from their command line.
+
+## Features
+
+- **Dataset Operations**: Create, read, write, and delete datasets
+- **USS File Access**: Create, read, write, and delete files and directories in Unix System Services
+- **Job Management**: Submit, view, cancel, hold, and release jobs on z/OS
+
+## Minimum Requirements
+
+- Zowe CLI v8.0.0 (or newer)
```

Comment:
Since we mention VS Code as a prereq for the VSCE, should we also mention Node.js LTS here? I'd be ok with omitting it too - since Zowe CLI requires Node it should be implied.

Reply:
I was thinking along the same lines - if they have Zowe CLI installed, they also have Node.js 😋